src/Entities/Player.cpp

```diff
@@ -2277,6 +2279,11 @@ bool cPlayer::LoadFromFile(const AString & a_FileName, cWorldPtr & a_World)
 	m_LifetimeTotalXp     = root.get("xpTotal",        0).asInt();
 	m_CurrentXp           = root.get("xpCurrent",      0).asInt();
 	m_IsFlying            = root.get("isflying",       0).asBool();
+	m_EnchantmentSeed     = root.get("enchantmentSeed", 0).asUInt();
```

Comment:
Can you pass the default, instead of 0, as the random number?

Reply:
I was trying to avoid an unused rand generation but I don't think it matters so yep :)